tests/cypress/integration/taxonomy.test.js

```diff
@@ -49,18 +50,26 @@ describe('Admin can create and update podcast taxonomy', () => {
 		cy.get('.row-title').should('have.text', 'Distributed');
 	});
 
-	it('Can delete taxonomy', () => {
-		cy.visit(
-			'/wp-admin/edit-tags.php?taxonomy=podcasting_podcasts&podcasts=true'
-		);
-		cy.get('.row-title').should('have.text', 'Distributed').click();
-		cy.url().should('contains', 'http://localhost:8889/wp-admin/term.php');
-		cy.on('window:confirm', () => true);
-		cy.get('.delete').click();
-		cy.url().should(
-			'contains',
-			'http://localhost:8889/wp-admin/edit-tags.php'
-		);
-		cy.get('.wp-list-table').should('contain.text', 'No podcasts found');
-	});
+	if (Cypress.env('HAS_BLOCKS_EDITOR')) {
+		it('Can delete taxonomy', () => {
+			cy.visit(
+				'/wp-admin/edit-tags.php?taxonomy=podcasting_podcasts&podcasts=true'
+			);
+			cy.get('.row-title').should('have.text', 'Distributed').click();
+			cy.url().should(
+				'contains',
```

Comment:
@dinhtungdu the 'contains' chainer [doesn't exist](https://docs.cypress.io/guides/references/assertions#Chai-jQuery).

There is 'contain' or 'contain.text'. Probably mixed up with [`cy.contains()`](https://docs.cypress.io/api/commands/contains) getter.

Could you please update it here (and probably other places not related to the current PR).

Reply:
Thanks for the head-up! I changed it to 'contain'.